### ISSUE TITLE
Bug - monitor-get/monitor-delete Erroring with NOT_FOUND

### ISSUE THREAD

**Issue by jim380** [msgbyte/tianji#130]
## Description
The following routes always return `NOT_FOUND` even with a valid `monitorId` that can be found at `/workspace/:workspaceId/monitor/all`:

- `GET /workspace/:workspaceId/monitor/:monitorId`
- `DELETE /workspace/:workspaceId/monitor/:monitorId`

```bash
{"message":"Not found","code":"NOT_FOUND"}
```

**Comment by moonrailgun** [comment]
![image](https://github.com/user-attachments/assets/8c6281d7-eda0-4344-8382-22340ff53e6b)

I think its should be work, maybe you can upgrade to latest version because we actually have this bug in old version.

**Comment by jim380** [comment]
> ![image](https://github.com/user-attachments/assets/8c6281d7-eda0-4344-8382-22340ff53e6b)
> 
> I think its should be work, maybe you can upgrade to latest version because we actually have this bug in old version.

`/monitor/all` does work, but the other two don't. Please see the issue description.

**Comment by moonrailgun** [comment]
yeah, we resolve this in https://github.com/msgbyte/tianji/issues/115

new api is `/workspace/{workspaceId}/monitor/{monitorId}/get` and `/workspace/{workspaceId}/monitor/{monitorId}/del`

**Comment by moonrailgun** [comment]
sorry, website api document is not up to date.
i will fix it
